%YAML:1.0
---
CameraExtrinsicMat: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 1., 0., 0., 0., 0., 1., 0., 0., 0., 0., 1., 0., 0., 0., 0.,
       1. ]
CameraMat: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.6150777879652162e+02, 0., 2.9948908821305247e+02, 0.,
       7.5981171834971371e+02, 1.8891527708367954e+02, 0., 0., 1. ]
DistCoeff: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [ -8.1107652432606769e-01, 3.5668344051746037e+00,
       -4.6632565323986097e-03, 1.3154602058187997e-02,
       -9.0310552957347294e+00 ]
ImageSize: [640, 360]
Reprojection Error: 0.0
DistModel: plumb_bob